src/___tests___/statusUpdates.test.js

```diff
@@ -0,0 +1,24 @@
+import { updateTaskStatus } from '../statusUpdates.js';
+import { saveTasks } from '../taskFunctions.js';
+
+jest.mock('../taskFunctions', () => ({
+  saveTasks: jest.fn(),
+}));
+
+describe('updateTaskStatus', () => {
+  afterEach(() => {
+    saveTasks.mockClear();
+  });
+
+  test('It updates the status and order of the task into the list', () => {
+    const tasks = [
+      { description: 'Tarea 1', completed: false },
+      { description: 'Tarea 2', completed: true },
+    ];
+
+    updateTaskStatus(tasks, 0, true);
+
+    expect(tasks[0].completed).toBe(true);
+    expect(saveTasks).toHaveBeenCalledWith(tasks);
+  });
```

Comment:
- [ ] Great job writitng your tests, remember that you also need to test your "Clear all completed" button. Kindly add this test to your files.

   ![image](https://github.com/gdvl777/to_do_list_ok/assets/107952724/d4e80672-997f-4ee0-acb7-6d81f22f7a08)